cornichon-kafka/src/main/scala/com/github/agourlay/cornichon/kafka/KafkaDsl.scala

```diff
@@ -36,17 +36,12 @@ trait KafkaDsl {
     }
   )
 
-  def read_from_topic(topic: String, amount: Int, targetKey: Option[String] = None, timeout: Int = 500) = EffectStep.fromAsync(
+  def read_from_topic(topic: String, amount: Int = 1, timeout: Int = 500) = EffectStep.fromAsync(
     title = s"reading the last $amount messages from topic=$topic",
-    effect = s ⇒ readFromTopic(topic, targetKey.getOrElse(topic), amount, timeout, s)(buildConsumerRecordJsonProjection(v ⇒ s""" "$v" """))
+    effect = s ⇒ readFromTopic(topic, amount, timeout, s)
```

Comment:
the indirection through the private `readFromTopic` is not necessary anymore since the Json alternative is gone